libs/sdk-core/src/logger.rs

```diff
@@ -0,0 +1,321 @@
+use crate::models::LevelFilter as BindingLevelFilter;
+use crate::{LogEntry, LogStream};
+use anyhow::{anyhow, Result};
+use chrono::Local;
+use env_logger::{Builder, Logger, Target};
+use flutter_rust_bridge::StreamSink;
+use lazy_static::lazy_static;
+use log::{
+    max_level, set_boxed_logger, set_max_level, warn, LevelFilter, Log, Metadata, Record,
+    STATIC_MAX_LEVEL,
+};
+use parking_lot::RwLock;
+use std::fs::OpenOptions;
+use std::io::Write;
+
+use std::sync::Once;
+
+/* env_logger */
+
+const ENV_LOGGER_FILTER: &str = r#"
+debug,
+breez_sdk_core::backup=info,
+breez_sdk_core::breez_services=info,
+breez_sdk_core::input_parser=warn,
+breez_sdk_core::persist::reverseswap=info,
+breez_sdk_core::reverseswap=info,
+gl_client::node=info,
+gl_client::node::service=info,
+gl_client::tls=info,
+gl_client::scheduler=info,
+gl_client::signer=info,
+gl_client=debug,
+h2=warn,
+h2::client=info,
+h2::codec::framed_read=warn,
+h2::codec::framed_write=warn,
+h2::proto::connection=info,
+h2::proto::settings=info,
+hyper=warn,
+hyper::client::connect::dns=info,
+hyper::client::connect::https=info,
+lightning_signer=warn,
+lightning_signer::node=warn,
+reqwest=warn,
+reqwest::connect=warn,
+rustls=warn,
+rustls::anchors=warn,
+rustls::conn=warn,
+rustls::client::common=warn,
+rustls::client::hs=warn,
+rustls::client::tls13=warn,
+rustyline=warn,
+rusqlite_migration=warn,
+tower::buffer::worker=warn,
+vls_protocol_signer=warn,
+vls_protocol_signer::handler::HandlerBuilder::do_build=warn
+"#;
+
+fn init_env_logger(target: Option<Target>, filter_level: Option<LevelFilter>) -> Logger {
+    let mut binding = Builder::new();
+    let builder = binding
+        .parse_filters(ENV_LOGGER_FILTER)
+        .format(|buf, record| {
+            writeln!(
+                buf,
+                "[{} {} {}:{}] {}",
+                Local::now().format("%Y-%m-%d %H:%M:%S%.3f"),
```

Comment:
Let's use UTC if possible with ISO8601 format

Reply:
Good point! Done.

[Switch timestamp format from local to UTC in ISO 8601 format](https://github.com/breez/breez-sdk-greenlight/pull/874/commits/48c1ea809b4c3ffaa54a89cb7f61cc1f50b19e4a)